packages/components/grid/src/stories/pagination.stories.ts

```diff
@@ -61,34 +40,47 @@ export const Basic: Story = {
         }
         sl-paginator {
           flex: 1;
+          justify-content: center;
         }
       </style>
-      <sl-grid .items=${people2.slice(0, 10)}>
-        <sl-grid-column path="firstName"></sl-grid-column>
-        <sl-grid-column path="lastName"></sl-grid-column>
-        <sl-grid-column path="profession"></sl-grid-column>
-        <sl-grid-column path="status"></sl-grid-column>
-        <sl-grid-column path="membership"></sl-grid-column>
+      <p>
+        This is an example of how to paginate data in a grid. It uses the <code>ArrayListDataSource</code> with the
+        <code>pagination</code> option enabled. You can change the page it starts on by setting the
+        <code>page</code> option. The page size can be changed by setting the <code>pageSize</code> option.
+      </p>
+      <sl-grid .dataSource=${ds}>
+        <sl-grid-column grow="0" header="Nr." path="studentNumber"></sl-grid-column>
+        <sl-grid-column
+          grow="3"
+          header="Student"
+          .renderer=${avatarRenderer}
+          .scopedElements=${{ 'sl-avatar': Avatar }}
+        ></sl-grid-column>
+        <sl-grid-column ellipsize-text header="School" path="school.name"></sl-grid-column>
       </sl-grid>
       <div class="pagination">
-        <sl-paginator-status .totalItems=${people2.length}></sl-paginator-status>
-        <sl-paginator @sl-page-change=${onPageChange} .totalItems=${people2.length}></sl-paginator>
-        <sl-paginator-page-size
-          @sl-page-size-change=${onPageSizeChange}
-          page-sizes="[5,10,15]"
-        ></sl-paginator-page-size>
+        <sl-paginator-status .dataSource=${ds}></sl-paginator-status>
+        <sl-paginator .dataSource=${ds}></sl-paginator>
+        <sl-paginator-page-size .dataSource=${ds} page-sizes="[5,10,15,20]"></sl-paginator-page-size>
       </div>
     `;
   }
 };
 
-export const DataSource: Story = {
-  name: 'Array Data Source',
-  render: (_, { loaded: { people } }) => {
-    const ds = new ArrayListDataSource(people as Person[], { pagination: true });
-    ds.setPage(2);
-    ds.setPageSize(10);
-    ds.update();
+export const Filtering: Story = {
+  render: (_, { loaded: { students } }) => {
```

Comment:
When we are filtering the data, there should be an information above the grid what is the amount of total items?

Reply:
That is not part of this story. That will be added/implemented as part of the filter pattern.